dockhand-lru-registry/values.yaml

```diff
@@ -0,0 +1,49 @@
+registry:
+  image:
+      repository: registry
+      pullPolicy: IfNotPresent
+      tag: 2.7.1
+  cache:
+    enabled: true
+    emptyDir:
```

Comment:
how does `emptyDir` behave?  do the calls to get disk size work if it's a shared `emptyDir`?

or should we require persistence with a complete disk to itself?

Reply:
Probably should require a disk

Reply:
Added `--separate-disk` argument to allow for when you are using a shared disk